src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -102,7 +102,7 @@ object Run  {
   private case class EventStartTime(name: String, timestamp: DateTime)
 
   def getEventList(op: Operation): Seq[ExecutionEventEntry] = {
-    val starterEvents = eventIfExists("createTime", op, "chilling out, maxing") ++ eventIfExists("startTime", op, "relaxing all cool")
+    val starterEvents = eventIfExists("createTime", op, "waiting for quota") ++ eventIfExists("startTime", op, "reticulating splines")
```

Comment:
I'm not 100% sure. It's whatever happens between quota becoming available and the VM actually starting up. Any good naming idea? Mustering Resources? Assigning Quota?